README.md

```diff
@@ -520,6 +537,25 @@ Each validator can accept an object with a `message` key or - in the case where
 you don't have to specify anything other than a validation message - just a
 string with the error message.
 
+##### `initialValues: object`
+
+The `initialValues` object lets you specify the initial values of the form
+fields. These values are available from the `fields` argument in the `children`
+function, which lets you control your form fields.
```

Comment:
Can you remove the same text from line 432-436?

Reply:
I removed all of the duplication and added a note to `FormValidation` to look at `Form` & `Validation` for the props. Let me know if this is acceptable.

Reply:
Yeah that's fine.